### ISSUE TITLE
Possible bug in 2.0.1?

### ISSUE THREAD

**Issue by swuecho** [milaboratory/mixcr#172]
```
➜  mixcr_trb git:(master) ✗ ls
test.fastq
➜  mixcr_trb git:(master) ✗ ~/app/mixcr-2.0.1/mixcr align --chains TRB test.fastq test.vdjca
Reference library: repseqio.v1.1:9606 (1353156c3504ab36ef955667a3e344d1)
Alignment: 0.2%
Alignment: 10.7%  ETA: 00:00:42
Alignment: 23.7%  ETA: 00:00:17
Alignment: 36.4%  ETA: 00:00:15
Alignment: 48.9%  ETA: 00:00:12
Alignment: 61.6%  ETA: 00:00:09
Alignment: 74.4%  ETA: 00:00:06
Alignment: 89.1%  ETA: 00:00:02
Alignment: 99.2%  ETA: 00:00:00
============= Report ==============
Analysis time: 25.82s
Total sequencing reads: 943395
Successfully aligned reads: 488614 (51.79%)
Alignment failed, no hits (not TCR/IG?): 69701 (7.39%)
Alignment failed because of absence of J hits: 381285 (40.42%)
Alignment failed because of low total score: 3795 (0.4%)
Overlapped: 0 (0%)
Overlapped and aligned: 0 (0%)
Overlapped and not aligned: 0 (0%)
[TRB] chains: 488614 (100%)
➜  mixcr_trb git:(master) ✗ ~/app/mixcr-2.0.1/mixcr assemble test.vdjca test.clns
Initialization: progress unknown
Assembling initial clonotypes: 13.2%
Assembling initial clonotypes: 30.9%  ETA: 00:00:03
Assembling initial clonotypes: 44.4%  ETA: 00:00:04
Assembling initial clonotypes: 55.1%  ETA: 00:00:04
Assembling initial clonotypes: 66.1%  ETA: 00:00:03
Assembling initial clonotypes: 77.1%  ETA: 00:00:02
Assembling initial clonotypes: 97.4%  ETA: 00:00:00
Mapping low quality reads: 5.7%
Mapping low quality reads: 18.1%  ETA: 00:00:06
Mapping low quality reads: 30.6%  ETA: 00:00:05
Mapping low quality reads: 41.2%  ETA: 00:00:05
Mapping low quality reads: 54%  ETA: 00:00:03
Mapping low quality reads: 68.2%  ETA: 00:00:02
Mapping low quality reads: 78.9%  ETA: 00:00:01
Mapping low quality reads: 91%  ETA: 00:00:00
Clustering: 0%
Clustering: 10.3%  ETA: 00:00:08
Clustering: 29.3%  ETA: 00:00:03
Clustering: 49.5%  ETA: 00:00:02
Clustering: 70.1%  ETA: 00:00:01
Clustering: 93.2%  ETA: 00:00:00
Building clones: 31.6%
Writing clones: 0%
============= Report ==============
Analysis time: 24.82s
Final clonotype count: 20882
Average number of reads per clonotype: 17.26
Reads used in clonotypes, percent of total: 360322 (38.19%)
Reads used in clonotypes before clustering, percent of total: 373043 (39.54%)
Number of reads used as a core, percent of used: 362977 (97.3%)
Mapped low quality reads, percent of used: 10066 (2.7%)
Reads clustered in PCR error correction, percent of used: 12721 (3.41%)
Reads pre-clustered due to the similar VJC-lists, percent of used: 0 (0%)
Reads dropped due to the lack of a clone sequence: 5296 (0.56%)
Reads dropped due to low quality: 333 (0.04%)
Reads dropped due to failed mapping: 109942 (11.65%)
Reads dropped with low quality clones: 0 (0%)
Clonotypes eliminated by PCR error correction: 1203
Clonotypes dropped as low quality: 0
Clonotypes pre-clustered due to the similar VJC-lists: 0
[TRB] chains: 20882 (100%)
```

align and assemble works fine. the problem is in exportClones step.

```
➜  mixcr_trb git:(master) ✗ ~/app/mixcr-2.0.1/mixcr exportClones --preset min test.clns test.txt
Exception in thread "main" java.lang.NullPointerException
        at com.milaboratory.mixcr.cli.ActionExportClones$CloneExportParameters.getFilter(ActionExportClones.java:178)
        at com.milaboratory.mixcr.cli.ActionExportClones.go0(ActionExportClones.java:61)
        at com.milaboratory.mixcr.cli.ActionExport.go(ActionExport.java:51)
        at com.milaboratory.cli.JCommanderBasedMain.main(JCommanderBasedMain.java:147)
        at com.milaboratory.mixcr.cli.Main.main(Main.java:113)

```

**Comment by swuecho** [comment]
I am happy to provide the 'test.fastq' file if necessary.

**Comment by dbolotin** [comment]
I confirm it is a bug! Thank you for reporting!

Fixed in 59675c5a130f158b7106b6ae3d080e26563f895a

Just released [fixed](https://github.com/milaboratory/mixcr/releases/tag/v2.0.2) version. You can also update using brew.

Please confirm that now it works as expected.

**Comment by swuecho** [comment]
works now. Thanks for quick fix！
